### ISSUE TITLE
BsNav and BsModalSimple throws `Cannot read property 'Symbol(TAG_COMPUTE)' of undefined`

### ISSUE THREAD

**Issue by basz** [kaliber5/ember-bootstrap#1132]
Should work

```
      <BsNav @type="tabs" as |nav|>
        <nav.item>
          <nav.link-to @route="index">"menu.orders"</nav.link-to>
        </nav.item>
        <nav.item>
          <nav.link-to @route="index">"menu.dossiers"</nav.link-to>
        </nav.item>
      </BsNav>
```

But does not
```
reference.js:121 Uncaught (in promise) TypeError: Cannot read property 'Symbol(TAG_COMPUTE)' of undefined
    at MonomorphicTagImpl.[TAG_COMPUTE] (reference.js:121)
    at MonomorphicTagImpl.[TAG_COMPUTE] (reference.js:121)
    at MonomorphicTagImpl.[TAG_COMPUTE] (reference.js:121)
    at MonomorphicTagImpl.[TAG_COMPUTE] (reference.js:121)
    at MonomorphicTagImpl.[TAG_COMPUTE] (reference.js:121)
    at update (reference.js:166)
    at TryOpcode.didInitializeChildren (runtime.js:4121)
    at VM.exit (runtime.js:5204)
    at Object.evaluate (runtime.js:698)
    at AppendOpcodes.evaluate (runtime.js:73)
```
The menu *is* rendered. Seems to come from MonomorphicTagImpl which is part of glimmer/component. The containing component is indeed a glimmer./component (1.0.0)

(using e-b 4.rc2)
(just tried out glimmer v2.beta7, no change)

**Comment by simonihmig** [comment]
@basz thanks for reporting! Just came across the same exception, but in my case using `<BsModalSimple>`. Other components don't suffer from this issue AFAICT.

Quite strange, especially that this was not uncovered in any test. 🤔

This does not happen for me w/ `4.0.0-rc.1`, are you able to confirm?

**Comment by simonihmig** [comment]
More weirdness:
* using dummy app, `/fastboot/modal` renders in FastBoot, fails later when rendered by Ember with that error
* no errors when running in integration tests

**Comment by basz** [comment]
rc1 did not have this error iirc.

**Comment by simonihmig** [comment]
This should be fixed in the just released 4.0.0-rc.3!
